.woodpecker/docker.yml

```diff
@@ -201,6 +201,7 @@ steps:
       dockerfile: docker/Dockerfile.agent.multiarch
       platforms: *platforms_preview
       tag: pull_${CI_COMMIT_PULL_REQUEST}
+      build_args: CI_COMMIT_SHA
```

Comment:
need to test if that's enough of if it has to be `CI_COMMIT_SHA=${CI_COMMIT_SHA}` ... localy the buildarg did pass an env through if no `=` was set ... might have a look into the docs

Reply:
well could not find anything so I switched to the documented `K=V` aproach